modules/sharedaddy/amp-sharing.css

```diff
@@ -0,0 +1,110 @@
+amp-social-share {
+	border-radius: 50%;
+	color: #fff !important;
+	-webkit-border-radius: 50%;
+	padding: 7px;
+	position: relative;
+	line-height: 1;
+	width: 32px;
+	height: 32px;
+	margin: 0 5px 5px 0;
+}
+
+amp-social-share::before {
+	display: inline-block;
+	font: normal 18px/1 'social-logos';
+	position: relative;
+	top: 1px;
+	vertical-align: top;
+	text-align: center;
+}
+
+amp-social-share[type='print'] {
+	background: #e9e9e9;
+}
+
+amp-social-share[type='print']::before {
+	content: '\f469';
+	color: #656565 !important;
```

Comment:
Unfortunately not, this is the output without it:

<img width="255" alt="Screen Shot 2020-04-30 at 3 48 33 PM" src="https://user-images.githubusercontent.com/49400/80718196-21198880-8afa-11ea-962c-7019b3a9c179.png">

Reply:
Is it getting overridden by an AMP style or some other style in Jetpack? What selector is overriding it? An alternative to `!important` would be a more-specific selector.

Reply:
@westonruter @jeherve The style rules here were copied from the current non-AMP stylesheet for consistency between the two share bars on AMP and non-AMP. The original stylesheet had the `!important` rules, i.e. for reddit: https://github.com/Automattic/jetpack/blob/master/modules/sharedaddy/sharing.css#L642

I _believe_ they would have been added intentionally to avoid theme conflicts on the styles. However,  I'm not apposed to removing them for the AMP stylesheet. Here's a commit that removes the parent level important rule and the rules for reddit and print. https://github.com/Automattic/jetpack/pull/15497/commits/2a1ffca14212c663b36647f2ad80dac94a23997f